FileIO/FEFLOWInterface.cpp

```diff
@@ -139,6 +141,54 @@ MeshLib::Mesh* FEFLOWInterface::readFEFLOWFile(const std::string &filename)
 			line_stream.clear();
 		}
 		//....................................................................
+		// ELEMENTALSETS
+		else if (line_string.compare("ELEMENTALSETS") == 0)
+		{
+			unsigned mode = 0;
+			std::string str_nodeList;
+			int pos_prev_line = 0;
+			while (true)
```

Comment:
Could be `do { ... } while (mode != 0)`, or change the `mode = 0` lines below to `break` and make the mode variable local to the loop.